client/my-sites/activity/activity-log-confirm-dialog/index.jsx

```diff
@@ -38,6 +42,47 @@ const ActivityLogConfirmDialog = ( {
 
 			<div className="activity-log-confirm-dialog__highlight">{ children }</div>
 
+			{ config.isEnabled( 'rewind/partial-restores' ) && (
+				<Card className="activity-log-confirm-dialog__partial-restore-settings">
+					<p>
+						<strong>
+							{ notice
+								? translate( 'Partial Restore Settings (A8C Only)' )
+								: translate( 'Partial Download Settings (A8C Only)' ) }
+						</strong>
+					</p>
+					<p>
+						{ notice
+							? translate( 'Include the following things in this restore:' )
+							: translate( 'Include the following things in this download:' ) }
+					</p>
+					<FormLabel>
+						<FormCheckbox name="themes" onChange={ onSettingsChange } defaultChecked />
+						{ translate( 'WordPress Themes' ) }
+					</FormLabel>
+					<FormLabel>
+						<FormCheckbox name="plugins" onChange={ onSettingsChange } defaultChecked />
+						{ translate( 'WordPress Plugins' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Unlimited WordPress plugins.' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind-partial-restores)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).